cmd/minikube/cmd/start.go

```diff
@@ -197,6 +201,11 @@ func initDriverFlags() {
 
 	// hyperv
 	startCmd.Flags().String(hypervVirtualSwitch, "", "The hyperv virtual switch name. Defaults to first found. (only supported with HyperV driver)")
+
+	// kic
+	startCmd.Flags().String(kicImg, "", "override the kic image. (Only to be used with on with kic driver) ")
```

Comment:
Could we use `--image`? I would prefer not to have `kic` as a second brandname that users need to learn about.